packages/docs/docs/home.md

```diff
@@ -10,13 +10,13 @@ import HomepageCallout from '@site/src/components/HomepageCallout'
 <section className="homepage-grid">
     <HomepageCallout title="Get Started" body="Write your first medical application in 5 minutes" linkText="Read more" linkRef="./docs/tutorials" />
     <HomepageCallout title="API Docs" body="Reference documentation about Medplum's client API for reading and writing healthcare data" linkText="Read more" linkRef="./docs/api" />
-    <HomepageCallout title="Solutions" body="See how to apply Medplum against your healthcare problem" linkText="Read more" linkRef="./solutions" />
+    <HomepageCallout title="Medplum EHR" body="A strealined EHR that's easy to implement" linkText="Read more" linkRef="./solutions/medplum-ehr" />
```

Comment:
I'm ok with this, but this is not what I had imagined during offline discussions.  My memory was that we would keep the "Solutions" title, but include "Medplum EHR" in the body.